kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/log/LogBatchCommand.scala

```diff
@@ -55,15 +55,17 @@ class LogBatchCommand(
 
       withKyuubiInstanceRestClient(kyuubiRestClient, kyuubiInstance) { kyuubiInstanceRestClient =>
         val kyuubiInstanceBatchRestApi: BatchRestApi = new BatchRestApi(kyuubiInstanceRestClient)
+
+        def getOperationLogOnce(): OperationLog = {
```

Comment:
prefer to use "fetch" or "retrieve" to distinguish w/ "getter", and it not fetch but also print the log, the method name should mention that, or introduce a param to control the display behavior.